src/components/BlackjackGame.tsx

```diff
@@ -55,8 +54,9 @@ function BlackjackGame(props: Props) {
   function MakeCards(hand) {
     const displayHand : JSX.Element[] = [];
     for (let i = 0; i < hand.length; i += 2) {
+      const link = ['https://deckofcardsapi.com/static/img/', hand[i], hand[i + 1], '.png'].join('');
       displayHand.push(
-        <BlackjackCard suit={hand[i + 1]} value={hand[i]} />,
```

Comment:
We could keep BlackjackCard and just pass in the hand so that we can make changes easier later. And the blackjack card could just display the image.